### PR TITLE
Newest client preferences

### PR DESCRIPTION
**Pull request by richards2k**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,13 +14,23 @@
     "@fortawesome/free-brands-svg-icons": "^5.13.0",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/vue-fontawesome": "^0.1.9",
+    "@popperjs/core": "^2.3.3",
```
Comment (shifuchen98) [inline package.json:17]: Is this used?

File: package.json [hunk 2]
```diff
@@ -14,13 +14,23 @@
...
+    "babel-eslint": "^10.0.3",
     "core-js": "^3.6.4",
     "express": "^4.17.1",
     "jspdf": "^1.5.3",
     "jspdf-autotable": "^3.5.3",
     "leancloud-storage": "^4.5.3",
     "leanengine": "^3.6.0",
     "vue": "^2.6.11",
+    "vue-js-modal": "^1.3.35",
+    "vue-js-popover": "^1.1.8",
```
Comment (shifuchen98) [inline package.json:33]: And this?

File: package.json (hunk 1 above)
Comment (richards2k) [reply package.json:17]: Not anymore. I used it before only to find it not working the way I wanted. This can be deleted

File: package.json (hunk 2 above)
Comment (richards2k) [reply package.json:33]: I'm using the vue-js-modal only, so you can delete the vue-js-popover, and keep the vue-js-modal

File: package.json (hunk 1 above)
Comment (shifuchen98) [reply package.json:17]: Could you delete it?

Comment (richards2k) [reply package.json:17]: okay

Comment (richards2k) [reply package.json]: I deleted the code in my branch already. Do i have to make a new pull request?

Comment (shifuchen98) [reply package.json]: No.

**Pull request merged**